pkgs/tools/misc/ugtrain/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, autoconf, automake, binutils, glib, libtool, pkg-config, scanmem, fetchFromGitHub }:
```

`glib` not used. Also, it's somewhat common to use lines to separate arguments, and putting the "native" deps first:

```suggestion
{ stdenv
, fetchFromGitHub
, autoreconfHook
, pkg-config
, scanmem
}:
```